### PR TITLE
Insert mobile version  for work section and about section

### PR DESCRIPTION
**Pull request by NatanimA**
-->Inserted Mobile version for work section and about section
-->Included pictures in work section
--->inserted logos in languages
--->Fixed issues of non responsivness
---->finished mobile version for about and work section
--->edited the readme

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -267,8 +394,100 @@ nav {
...
+  }
+
+  .project {
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: flex-start;
+    width: 80vw;
+    height: auto;
+    padding: 24px;
+  }
```
Comment (awais-amjed) [inline css/style.css]: Please make use of flex-box to make your "projects div" responsive 🤗

File: css/style.css [hunk 2]
```diff
@@ -267,8 +394,100 @@ nav {
...
 
-  .connect {
+  #work {
+    align-items: center;
+  }
+
+  .project {
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: flex-start;
```
Comment (awais-amjed) [inline css/style.css]: [OPTIONAL] Kindly note that these lines have no effect unless you use `display: flex;` 😃

File: index.html [hunk 3]
```diff
@@ -54,9 +52,129 @@ <h2 class="connect">LET’S CONNECT</h2>
...
+                        <li class="lang">html</li>
+                        <li class="lang">css</li>
+                        <li class="lang">javaScript</li>
+                        <li class="lang">bootstrap</li>
+                    </ul> <button class="enabled-btn">See Project</button>
+                </div>
             </div>
-            
         </section>
+        <section id="about">
```
Comment (awais-amjed) [inline index.html:128]: Please make sure that the about myself section also follows the design 😀

Your design:
image

Figma Design:
image

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @NatanimA , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No Linter issues ✔️ 
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 
- [x] Good use of Semantics ✔️ 

gif

## Required Changes ♻️

- [ ] Please make sure that your design fits in the [mobile view](https://www.browserstack.com/guide/view-mobile-version-of-website-on-chrome) for the whole website. Kindly make use of `flex` to make your design responsive to different screen sizes 😃
  - _Mobile version should be responsive until the desktop screen size._

_Here is a comparison. Your design:_
image

_Figma Design:_
image

- [ ] Kindly import the Poppins font that you are trying to use on your website. It's better to import it at the top of your `style.css` file 🤗
  -  _Custom fonts won't work unless you import them._

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (NatanimA) [reply css/style.css]: 😉 forgot to add it lol
can you check now I fixed it.

File: css/style.css [hunk 4]
```diff
@@ -174,101 +176,157 @@ nav {
...
+  border: 2px solid #dfe1e6;
+  border-radius: 18px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: box-shadow 0.5s ease-in-out;
+}
+
+.project-picture {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
```
Comment (phelian23) [inline css/style.css:366]: - [ ] As requested by the previous reviewer, kindly make sure your design match as much as possible the template design. kindly check the font color of the project title and description.
- [ ] Also the margins of the title and description.
Your design
Capture
Figma template
figma2

File: css/style.css [hunk 5]
```diff
@@ -174,101 +176,157 @@ nav {
...
+  list-style-type: none;
+  font-size: 21px;
+  font-weight: 450;
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  cursor: pointer;
+  padding: 23px 42px 23px 12px;
+}
```
Comment (phelian23) [inline css/style.css:280]: - [ ]  As requested by the previous reviewer, kindly make sure your design match as much as possible the template design. Kindly take a look at the margins between the languages.
Your design
Capture1
Figma design
figma

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @NatanimA,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 5 above)
Comment (NatanimA) [reply css/style.css:280]: Ok I fixed it ,can you check now?
😄

**Review by aimemalaika — APPROVED** [review]
Hi @NatanimA , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**